aesthetic/avg_autotags.py

```diff
@@ -23,51 +23,80 @@
     # e.g. 'Upper Lawrenceville': Counter({'indoor': 28.342, 'text': 20.451, ...})
     nghds_autotags = collections.defaultdict(collections.Counter)
 
+    # this is the final output dict with neighborhood: 'num_photos': x , 'autotags_90plus_minusbaseline': [{'autotag': x, 'score': x, 'examples': x}, ]
+    nghd_autotags_urls = dict()
     overall_autotags_90plus = collections.Counter()
     for row in csv.reader(open(args.yfcc_autotags_file)):
-        photo_id, nsid, lat, lon = row[0:4]
+        photo_id, nsid, lat, lon, url = row[0:5]
         nghd = pgh_pointmap[lat, lon]
-
-        autotags_90plus = ast.literal_eval(row[4])
-        autotags = row[5].split(',')
+        # autotags_90plus is the autotags for each pic with >=0.9 score
+        autotags_90plus = ast.literal_eval(row[5])
+        autotags = row[6].split(',')
```

Comment:
Overall comment: don't be shy about deleting code! For example, this `autotags` variable here is not needed anymore, so you can delete a lot of code by removing it.